src/web/app/components/response-time-series-chart/response-time-series-chart.component.ts

```diff
@@ -29,20 +29,17 @@ export class ResponseTimeSeriesChartComponent implements OnInit {
 
   ngOnInit(): void {
     this.interval.pipe(
-        switchMap((value: number) => interval(value * 60 * 1000)),
+        switchMap((value: number) => interval(value * 1000)),
```

Comment:
1 second?

Reply:
Maybe make it 10 seconds or so.
Also, do add a `refresh()` call in this function to force the data init.